### PR TITLE
included devfile converter in migration docs

### PR DESCRIPTION
**Pull request by jc-berger**
Fixes https://github.com/devfile/api/issues/722 

The issue asked to reference this link: https://www.npmjs.com/package/@eclipse-che/devfile-converter 

I know Red Hat discourages to link to external webpages. So I included this content as new documents. Please let me know what you think, thanks!

### CODE REVIEW

File: docs/modules/user-guide/partials/proc_installing-the-devfile-converter.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[id="proc_installing-the-devfile-converter_{context}"]
+= Installing the devfile-converter
+
+[role="_abstract"]
+As you migrate an existing v1.x devfile to a v2.x devfile, consider using the devfile-converter.
```
Comment (rolfedh) [inline docs/modules/user-guide/partials/proc_installing-the-devfile-converter.adoc:5]:
```suggestion
To migrate a devfile from version 1.x to 2.x, install and use the devfile-converter.
```

File: docs/modules/user-guide/partials/proc_installing-the-devfile-converter.adoc [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+----
+$ import * as devfileConverter from '@eclipse-che/devfile-converter';
+----
++
+----
+$ async function v2ToV1(devfileV2: Devfile, externalContentAccess?: (filename: string) => Promise<string>): Promise<che.workspace.devfile.Devfile>
+----
++
+----
+$ async function v1ToV2(devfileV1: che.workspace.devfile.Devfile): Promise<Devfile>
```
Comment (rolfedh) [inline docs/modules/user-guide/partials/proc_installing-the-devfile-converter.adoc:20]: Is this a conversion step? If so, would it make sense to have separate the installation and conversion steps?

**Review by rolfedh — APPROVED** [review]
Made some minor suggestions. Otherwise, LGTM.

File: docs/modules/user-guide/partials/proc_installing-the-devfile-converter.adoc (hunk 2 above)
Comment (jc-berger) [reply docs/modules/user-guide/partials/proc_installing-the-devfile-converter.adoc:20]: Made the change. I'll see if it looks good and makes sense to SME reviewers!

**Review by Srivaralakshmi — APPROVED** [review]
@jc-berger Nice work! LGTM :)

**Comment by jc-berger** [comment]
Please go to the new PR for current work on this concern: https://github.com/devfile/docs/pull/127

**Pull request closed without merging**